sql/core/src/test/scala/org/apache/spark/sql/execution/metric/SQLMetricsSuite.scala

```diff
@@ -788,6 +792,28 @@ class SQLMetricsSuite extends SharedSparkSession with SQLMetricsTestUtils
     }
   }
 
+  test("SPARK-34399: Add job commit duration metrics for DataWritingCommand") {
+    withSQLConf(SQLConf.FILE_COMMIT_PROTOCOL_CLASS.key ->
+      "org.apache.spark.sql.execution.metric.CustomFileCommitProtocol") {
+      withTable("t", "t2") {
+        sql("CREATE TABLE t(id STRING) USING PARQUET")
+        sql("INSERT INTO TABLE t SELECT 'abc'")
+        sql("INSERT INTO TABLE t SELECT 'abc'")
+        sql("INSERT INTO TABLE t SELECT 'abc'")
+        sql("CREATE TABLE t2(id STRING) USING PARQUET")
+        val df = sql("INSERT INTO TABLE t2 SELECT * FROM  t")
```

Comment:
can we simplify the test and just write `INSERT INTO TABLE t SELECT 'abc'`?

Reply:
Done